lib/chef/knife.rb

```diff
@@ -32,12 +32,12 @@
 require_relative "http/http_request"
 require_relative "http"
 require "pp" unless defined?(PP)
-require_relative "dist"
+require "chef-utils"
```

Comment:
This should probably be the following, correct?
```suggestion
require "chef-utils/dist"
```